src/sentry/snuba/events.py

```diff
@@ -45,6 +45,7 @@ class Columns(Enum):
         event_name="group_ids",
         transaction_name="group_ids",
         discover_name="group_ids",
+        issue_platform_name=None,
```

Comment:
Do we need to add `None` explicitly here? It's the default value for this field on `Column` anyway, not sure this actually changes how it works?

Reply:
Yeah, you're right. We might have to add a tag 'blacklist' to the dataset for any known columns that shouldn't filter on tags. Hmm